src/ol/layer/VectorTile.js

```diff
@@ -165,11 +166,4 @@ class VectorTileLayer extends BaseVectorLayer {
 }
 
 
-/**
- * Return the associated {@link module:ol/source/VectorTile vectortilesource} of the layer.
- * @function
- * @return {import("../source/VectorTile.js").default} Source.
- * @api
- */
-VectorTileLayer.prototype.getSource;
```

Comment:
Same question as above. What about API docs? If we loose API docs by removing this, we should have a JSDoc plugin that brings this information back.